oat-sa/extension-tao-mediamanager
# Feature/AUT-35/Adding rich passages styling to an item

## Relates to: _[AUT-35](https://oat-sa.atlassian.net/browse/AUT-35)_

### Changes
* Added an endpoint to get the stylesheet list for the necessary passage;
* Added an endpoint to get the stylesheet for the necessary passage;
* Code refactoring.

### How to test
1. Create a passage;
2. Style this passage and save;
3. Go to `[[ HOST ]]/taoMediaManager/SharedStimulusStyling/getStylesheets?uri=[[ PASSAGE_URI ]]` and check response - it should contain a list of related stylesheet files (names only);
4. Go to `[[ HOST ]]/taoMediaManager/SharedStimulusStyling/loadStylesheet?uri=[[ PASSAGE_URI ]]&stylesheet=[[ STYLESHEET_NAME ]]` and check response - it should contain a css content (`text/css`).

---

### Where to test
Environment: _http://test-shpran.playground.kitchen.it.taocloud.org:41481/_
Credentials: _admin / admin_

---

### Companion PRs
_[extension-tao-itemqti#1747](https://github.com/oat-sa/extension-tao-itemqti/pull/1747)_
